docs/user/tutorials/01-40-expose-workload/v2alpha1/01-43-expose-workload-short-host-name.md

```diff
@@ -0,0 +1,127 @@
+# Expose a Workload with Short Host Name
+
+This tutorial shows how to expose an unsecured instance of the HTTPBin Service using short host name without domain and call its endpoints.
+
+> [!WARNING]
+>  Exposing a workload to the outside world is a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [JWT](../../01-50-expose-and-secure-a-workload/v2alpha1/01-52-expose-and-secure-workload-jwt.md).
+
+## Prerequisites
+
+* [Deploy a sample HTTPBin Service](../../01-00-create-workload.md).
+
+## Steps
+
+### Expose Your Workload
+
+1. Export the necessary values as environment variables:
+
+  ```bash
+  export DOMAIN_TO_EXPOSE_WORKLOADS={KYMA_DOMAIN_NAME}
+  export GATEWAY=$NAMESPACE/httpbin-gateway
+  ```
+
+2. Create custom Gateway specifying host domain:
+
+<!-- tabs:start -->
+#### **Kyma Dashboard**
+
+* Go to **Istio > Gateways** and select **Create**.
+* Provide the following configuration details:
+    - **Name**: `httpbin-gateway`
+    - **Namespace**: `{NAMESPACE_NAME}`
+    - In the `Servers` section, select **Add**. Then, use these values:
+      - **Port Number**: `80`
+      - **Name**: `http`
+      - **Protocol**: `HTTP`
+    - Use `httpbin.{KYMA_DOMAIN_NAME}` as **Host**.
```

Comment:
Why `httpbin.{KYMA_DOMAIN_NAME}`? Shouldn't it be `*.{DOMAIN_TO_EXPOSE_WORKLOADS}` ?

Reply:
No relevant, since we use it from the referenced guide for the custom domain, this section will be removed.